contracts/utils/Strings.sol

```diff
@@ -42,7 +42,7 @@ library Strings {
      * @dev Converts a `int256` to its ASCII `string` decimal representation.
      */
     function toString(int256 value) internal pure returns (string memory) {
-        return string(abi.encodePacked(value < 0 ? "-" : "", toString(SignedMath.abs(value))));
+        return string.concat(value < 0 ? "-" : "", toString(SignedMath.abs(value)));
```

Comment:
This is something we have planed for 5.0 but that we decided not to do so far for a simple reason:

`bytes.concat` was introduced in 0.8.4, and `string.concat` was properly introduced in 0.8.12. If we do this change, we should change the pragma for this file from `^0.8.0` to `^0.8.12` ... which is a breaking change.

Reply:
Agree with this, let's remove this change and add it to the list of new Solidity features to use in 5.x.

Reply:
Thanks for flagging this, I had no idea. I just reverted that particular change and I'll circle this back with the audit team